### PR TITLE
[RAPPS][CONSRV] French translation update

### PR DESCRIPTION
**Pull request by KyleKatarn31**
## Purpose

- Missing French translation for some new items

## Proposed changes

- Add missing items in French translation

### CODE REVIEW

**Comment by KyleKatarn31** [comment]
@HBelusca @sdever  : for review

File: base/applications/rapps/lang/fr-FR.rc [hunk 1]
```diff
@@ -212,8 +212,8 @@ BEGIN
     IDS_MISMATCH_CERT_INFO "Certificat inconnu:\nSujet: %s\nEmetteur: %s\nVoulez-vous continuer ?"
     IDS_UNABLE_PATH "Format de chemin invalide."
     IDS_APP_DISPLAY_DETAILS "details"
```
Comment (HBelusca) [inline base/applications/rapps/lang/fr-FR.rc:214]:
```suggestion
    IDS_APP_DISPLAY_DETAILS "détails"
```

File: base/applications/rapps/lang/fr-FR.rc [hunk 2]
```diff
@@ -254,12 +254,12 @@ END
...
-    IDS_CMD_NEED_FILE_NAME "Error: option %1 expects a file name.\n"
-    IDS_CMD_NEED_PARAMS "Error: option %1 expects one or more parameters.\n"
-    IDS_CMD_INVALID_OPTION "Error: Unknown or invalid command line option specified.\n"
-    IDS_CMD_FIND_RESULT_FOR "Find result for %1:\n"
-    IDS_CMD_PACKAGE_NOT_FOUND "Failed to find package %1.\n"
-    IDS_CMD_PACKAGE_INFO "Information about package %1:\n"
+    IDS_CMD_USAGE "Usage : "
+    IDS_CMD_NEED_PACKAGE_NAME "Erreur : L'option %1 attend un ou plusieurs noms de paquets.\n"
+    IDS_CMD_NEED_FILE_NAME "Erreur : L'option %1 attent un nom de fichier.\n"
+    IDS_CMD_NEED_PARAMS "Erreur : L'option %1 attend un ou plusieurs paramètres.\n"
```
Comment (HBelusca) [inline base/applications/rapps/lang/fr-FR.rc:260]:
```suggestion
    IDS_CMD_NEED_PACKAGE_NAME "Erreur : L'option %1 nécessite un ou plusieurs noms de paquets.\n"
    IDS_CMD_NEED_FILE_NAME "Erreur : L'option %1 nécessite un nom de fichier.\n"
    IDS_CMD_NEED_PARAMS "Erreur : L'option %1 nécessite un ou plusieurs paramètres.\n"
```

File: base/applications/rapps/lang/fr-FR.rc [hunk 3]
```diff
@@ -254,12 +254,12 @@ END
...
+    IDS_CMD_INVALID_OPTION "Erreur : L'option de ligne de commande spécifiée est inconnue ou invalide.\n"
+    IDS_CMD_FIND_RESULT_FOR "Résultats pour %1:\n"
```
Comment (HBelusca) [inline base/applications/rapps/lang/fr-FR.rc:262]:
```suggestion
    IDS_CMD_FIND_RESULT_FOR "Résultats pour %1 :\n"
```

File: base/applications/rapps/lang/fr-FR.rc [hunk 4]
```diff
@@ -254,12 +254,12 @@ END
...
+    IDS_CMD_PACKAGE_NOT_FOUND "Impossible de trouver le paquet %1.\n"
+    IDS_CMD_PACKAGE_INFO "Information sur le paquet %1:\n"
```
Comment (HBelusca) [inline base/applications/rapps/lang/fr-FR.rc:264]:
```suggestion
    IDS_CMD_PACKAGE_INFO "Information sur le paquet %1 :\n"
```

**Comment by KyleKatarn31** [comment]
Thanks @HBelusca : All changes in commit

**Review by HBelusca — APPROVED** [review]

**Pull request merged**